src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyPrice = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+  const result = dailyPrice * days;
```

Comment:
But also, result is a bit confusing name of variable, proper variant would be something like `totalPrice`, as I commented above